src/main/java/com/bankofspring/model/Account.java

```diff
@@ -0,0 +1,95 @@
+package com.bankofspring.model;
+
+import java.util.List;
+
+/**
+ * Class to represent a bank account
+ * @author malcolmmurray
+ *
+ */
+public class Account {
+	
+	private String accountNumber;
+	private String accountDescription;
+	private Double accountAmount; //This is a Double object for now as this will be needed later for Hibernate
```

Comment:
Just FYI, you shouldn't use a Double to represent currency in Java ([see here](http://stackoverflow.com/questions/3730019/why-not-use-double-or-float-to-represent-currency))